docs/user-flows.md

```diff
@@ -1,262 +1,165 @@
+Historically, Lighthouse has analyzed the cold pageload of a page. Starting in 2022 (Lighthouse v10), it can analyze and report on the entire page lifecycle via "user flows".
 
-> Is that checkout form after you add an item to your cart accessible?
+#### You might be interested in flows if…
 
-> What is the Cumulative Layout Shift of my SPA page transition?
+* … you want to run Lighthouse on your whole webapp, not just the landing page.
+* … you want to know if all parts of the web experience are accessible (e.g. a checkout process).
+* … you want to know the Cumulative Layout Shift of my SPA page transition
 
-You want Lighthouse on a _flow_, not just a page load.
+In these cases, you want Lighthouse on a _flow_, not just a page load.
 
-This document describes how Lighthouse measures flows and offers recommendations on how to structure your own flow measurement with practical examples.
+## The three modes: Navigation, Timespan, Snapshot
 
-## Flow Building Blocks
+Lighthouse can now run in three modes: navigations, timespans, and snapshots. Each mode has its own unique use cases, benefits, and limitations. Later, you'll create a flow by combining these three core report types.
 
-Flow measurement in Lighthouse is enabled by three modes: navigations, timespans, and snapshots. Each mode has its own unique use cases, benefits, and limitations. Later, you'll create a flow by combining these three core report types.
+* **Navigation mode** analyzes a single page load. It should feel familiar as all Lighthouse runs prior to v9.6.0 were essentially in this mode. Even in v9.6.0 and onwards, it remains the default.
+* **Timespan mode** analyzes an arbitrary period of time, typically containing user interactions.
+* **Snapshot mode** analyzes the page in a particular state.
 
-### Navigation
+| | |
+|:---:|---|
+| Navigation <br> <img src="https://user-images.githubusercontent.com/39191/170560057-4f7cab97-ee6b-497f-8e18-1f84ba3616dd.png" height="96" width="141">  | **Use Cases** <br>  ✅ Obtain a Lighthouse Performance score and all performance metrics.<br>✅ Assess Progressive Web App capabilities.<br><br> **Limitations** <br>  🤔 Cannot analyze form submissions or single page app transitions.<br>🤔 Cannot analyze content that isn't available immediately on page load. |
+| Timespan <br> <img src="https://user-images.githubusercontent.com/39191/170560049-a6b630a2-187a-4233-9f8b-8d7fb276f51e.png" height="96" width="141">  | **Use Cases** <br> ✅ Measure layout shifts and JavaScript execution time over a timerange including interactions.<br>✅ Discover performance opportunities to improve the experience for long-lived pages and SPAs. <br>✅ Quantify single page app transitions.<br><br> **Limitations** <br>  🤔 Does not provide an overall performance score.<br>🤔 Cannot analyze moment-based performance metrics (e.g. Largest Contentful Paint).<br>🤔 Cannot analyze state-of-the-page issues (e.g. no Accessibility category) |
```

Comment:
I don't think we need "Quantify single page app transitions" because of the bullet point above it

Reply:
done